demo.html

```diff
@@ -9,12 +9,12 @@
 <body>
   <h1>speak.js</h1>
   <h2>Text-To-Speech on the Web</h2>
-  <form onsubmit="speak(text.value, { amplitude: amplitude.value, wordgap: workdgap.value, pitch: pitch.value, speed: speed.value }); return false">
+  <form onsubmit="speak(text.value, { amplitude: amplitude.value, wordgap: workggap.value, pitch: pitch.value, speed: speed.value }); return false">
```

Comment:
is this a typo fix, or something deeper?

if a type, shouldn't it be `wordgap`?